pandas/core/frame.py

```diff
@@ -1768,9 +1771,16 @@ def memory_usage(self, index=True, deep=False):
                             index=['Index']).append(result)
         return result
 
-    def transpose(self):
+    def transpose(self, *args, **kwargs):
         """Transpose index and columns"""
-        return super(DataFrame, self).transpose(1, 0)
+        # We don't want to pass in additional '*args' values
```

Comment:
you can do these comments, but they should all be in ``numpy_compat``. you are cluttering this code too much.